privacyidea/api/lib/utils.py

```diff
@@ -268,7 +268,10 @@ def get_priority_from_param(param):
     priority = {}
     for k, v in param.items():
         if k.startswith("priority."):
-            priority[k[len("priority."):]] = int(v)
+            try:
```

Comment:
you could simply use ``isinstance()`` in the if statement and check, if "v" is an int.

Please also add a short test in test_api_lib_utils.py